src/applications/coronavirus-screener/components/FormResult.unit.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { expect } from 'chai';
+import FormResult from './FormResult';
+
+describe('FormResult', () => {
+  it('renders FormResult component', async () => {
+    const formState = {
+      status: 'incomplete',
+    };
+    render(<FormResult formState={formState} />);
+
+    screen.debug();
+    /*
+<body>
+  <div>
+    <div
+      class="feature covid-screener-results covid-screener-results-incomplete"
+    >
+      <div
+        name="multi-question-form-result-scroll-element"
+      />
+      <div>
+        <div
+          data-testid="result-incomplete"
+        >
+          Please answer all the questions above.
+        </div>
+      </div>
+    </div>
+  </div>
+</body>
+*/
+    // Error: Unable to find an element by: [data-testid="result-incomplete"]
+    expect(await screen.findByTestId('result-incomplete')).to.exist();
```

Comment:
Could we update this to `expect(await screen.findByText('Please answer all the questions above.')).to.not.be.null;` and see if that works? 

We can test that this will fail if we change it as follows: `expect(await screen.findByText('Please answer all the questions above?')).to.not.be.null;`

Reply:
Fails with error:
```
Error: Unable to find an element with the text: Please answer all the questions above.. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

Reply:
Fails with test script: 
`yarn run test:unit --path src/applications/coronavirus-screener/components/FormResult.unit.spec.jsx`


Passes with manual command:
`BABEL_ENV=test ./node_modules/.bin/mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/**/FormResult.unit.spec.js?(x)'`

Reply:
can't get the RTL test to pass so i split it into another PR: https://github.com/department-of-veterans-affairs/vets-website/pull/14346